pkg/apis/core/validation/validation.go

```diff
@@ -2920,6 +2928,18 @@ func ValidateVolumeMounts(mounts []core.VolumeMount, voldevices map[string]strin
 			allErrs = append(allErrs, field.Invalid(idxPath.Child("mountPath"), mnt.MountPath, "must not already exist as a path in volumeDevices"))
 		}
 
+		// Disallow subPath/subPathExpr for OCI volumes
+		if utilfeature.DefaultFeatureGate.Enabled(features.OCIVolume) {
```

Comment:
Be sure to include this in the api doc / godoc of the volume source - `subPath mounts of this volume source are not supported.`.

Reply:
Done.